perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

```diff
@@ -0,0 +1,81 @@
+  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`"/>
+  <arg name="use_pointcloud_map" default="true" description="use pointcloud map in detection"/>
+
+  <!-- camera lidar fusion based detection-->
+  <group if="$(eval '&quot;$(var lidar_detection_mode)&quot;==&quot;camera_lidar_fusion&quot;')">
+    <include file="$(find-pkg-share perception_launch)/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml">
+      <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
+      <arg name="image_raw0" value="$(var image_raw0)"/>
+      <arg name="camera_info0" value="$(var camera_info0)"/>
+      <arg name="image_raw1" value="$(var image_raw1)"/>
+      <arg name="camera_info1" value="$(var camera_info1)"/>
+      <arg name="image_raw2" value="$(var image_raw2)"/>
+      <arg name="camera_info2" value="$(var camera_info2)"/>
+      <arg name="image_raw3" value="$(var image_raw3)"/>
+      <arg name="camera_info3" value="$(var camera_info3)"/>
+      <arg name="image_raw4" value="$(var image_raw4)"/>
+      <arg name="camera_info4" value="$(var camera_info4)"/>
+      <arg name="image_raw5" value="$(var image_raw5)"/>
+      <arg name="camera_info5" value="$(var camera_info5)"/>
+      <arg name="image_raw6" value="$(var image_raw6)"/>
+      <arg name="camera_info6" value="$(var camera_info6)"/>
+      <arg name="image_raw7" value="$(var image_raw7)"/>
+      <arg name="camera_info7" value="$(var camera_info7)"/>
+      <arg name="image_number" value="$(var image_number)"/>
+      <arg name="lidar_detection_model" value="$(var lidar_detection_model)"/>
+      <arg name="use_pointcloud_map" value="$(var use_pointcloud_map)"/>
+    </include>
+  </group>
+
+  <!-- lidar based detection-->
+  <group if="$(eval '&quot;$(var lidar_detection_mode)&quot;==&quot;lidar&quot;')">
+    <include file="$(find-pkg-share perception_launch)/launch/object_recognition/detection/lidar_based_detection.launch.xml">
+      <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
+      <arg name="lidar_detection_model" value="$(var lidar_detection_model)"/>
+      <arg name="use_pointcloud_map" value="$(var use_pointcloud_map)"/>
+    </include>
+  </group>
+
+  <!-- radar detection-->
+  <include file="$(find-pkg-share radar_tracks_msgs_converter)/launch/radar_tracks_msgs_converter.launch.xml">
+    <arg name="input/radar_objects" value="/sensing/radar/front_center/objects_raw" />
+    <arg name="input/odometry" value="/localization/kinematic_state" />
+    <arg name="output/radar_objects" value="radar/front_center/objects" />
+    <arg name="update_rate_hz" value="20.0" />
+    <arg name="use_twist_compensation" value="true" />
+  </include>
+
+  <!-- camera lidar radar fusion-->
+  <include file="$(find-pkg-share radar_fusion_to_detected_object)/launch/radar_object_fusion_to_detected_object.launch.xml">
+    <arg name="input/objects" value="lidar_objects" />
```

Comment:
`lidar_objects` is correct? I cannot find `lidar_objects`

Reply:
Sorry, I mistake merge to awf-latest.
I add `lidar_objects` output from LiDAR-based detection and Camera-LiDAR fusion.